ewah/hooks/oracle.py

```diff
@@ -52,6 +59,11 @@ def _adjust_sql(sql):
         return sql
 
     def _get_db_conn(self):
+        if not CX_ORACLE_AVAILABLE:
+            raise ImportError(
+                "cx_Oracle is not installed. Oracle support is only available on amd64 "
+                "architecture. Install with: pip install ewah[oracle]"
```

Comment:
Question (non-blocking): Does `pip install ewah[oracle]` actually work? How is that set up?